plugins/modules/web_infrastructure/django_manage.py

```diff
@@ -179,10 +189,23 @@ def _ensure_virtualenv(module):
     if venv_param is None:
         return
 
+    ack_venv_deprecation = module.params['ack_venv_creation_deprecation']
+
     vbin = os.path.join(venv_param, 'bin')
     activate = os.path.join(vbin, 'activate')
 
     if not os.path.exists(activate):
+        # In version 9.0.0, if the venv is not found, it should fail_json() here.
+        if not ack_venv_deprecation:
```

Comment:
Why not use
```suggestion
        if not module.params['ack_venv_creation_deprecation']:
```
here and remove the above assignment? That makes it easier to remove this in 9.0.0.

Reply:
Done